exercises/yacht/YachtTest.cs

```diff
@@ -0,0 +1,164 @@
+// This file was auto-generated based on version 1.0.0 of the canonical data.
+
+using Xunit;
+
+public class YachtTest
+{
+    [Fact]
+    // generated but commented out/renamed as it creates collision with the 'Yacht' class name
+    // public void Yacht()
+    public void Yacht_()
+    {
+        Assert.Equal(50, Yacht.Score(new[] { 5, 5, 5, 5, 5 }, "yacht"));
```

Comment:
For the second argument to score does it make sense to go with an Enum instead of a string? That would make sense to me since string parsing isn't a core concept in this exercise. Thoughts @ErikSchierboom and @ShamilS?

Reply:
Agreed @robkeim. Here is an example @ShamilS of how to render an enumeration: https://github.com/exercism/csharp/blob/master/generators/Exercises/TwoBucket.cs#L16 
I think the enumeration should probably be named `YachtCategory`, so that in the above example, the complete line would become:

```csharp
Assert.Equal(50, Yacht.Score(new[] { 5, 5, 5, 5, 5 }, YachtCategory.Yacht));
```

Reply:
Well, the exercise should be name `yacht-game`, not `yacht-dice-game`.

And the property should be set as follows I think:

```csharp
var category = canonicalDataCase.Properties["category"].ToString();
canonicalDataCase.Properties["category"] = new UnescapedValue($"YachtCategory.{category.Dehumanize()});
```